ui/js/controllers/perf/graphs.js

```diff
@@ -486,7 +486,17 @@ perf.controller('GraphsCtrl', [
                 });
 
                 //Closes popup on-click of "X"
-                $('#close-popup').bind("click", plotUnselected);
```

Comment:
Comments should have a space between `//` and the comment text. i.e. 

```js
// Closes popup on-click of "X"
```

Could you change this? (and the items below)